netbox/extras/migrations/0008_reports.py

```diff
@@ -18,7 +18,7 @@ def verify_postgresql_version(apps, schema_editor):
         with connection.cursor() as cursor:
             cursor.execute("SELECT VERSION()")
             row = cursor.fetchone()
-            pg_version = row[0].split()[1]
+            pg_version = re.match('^PostgreSQL (\d+\.\d+(\.\d+)?)', row[0])[1]
```

Comment:
Need to `import re` at the top

Reply:
🤦‍♂️ Sorry! Will update the PR. Do you care about having it rebased into a single commit?

Reply:
No worries, I'll squash & merge.